tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -646,3 +646,17 @@ test_i2c_as5600: as5600@54 {
 	compatible = "ams,as5600";
 	reg = <0x54>;
 };
+
+test_i2c_tmd2620: tmd2620@65 {
+	compatible = "ams,tmd2620";
+	reg = <0x65>;
```

Comment:
Here I used 0x65 as sequential address. This would be the right address if the entry before had the correct address.

Reply:
Opened a PR to correct the address of the entry before the tmd2620: #54264